DuckDuckGo/AutofillPixelReporter.swift

```diff
@@ -33,29 +33,34 @@ final class AutofillPixelReporter {
     var autofillOnboardedUser: Bool
 
     private let statisticsStorage: StatisticsStore
-    private let secureVault: (any AutofillSecureVault)?
+    private var secureVault: (any AutofillSecureVault)?
 
     enum EventType {
         case fill
         case searchDAU
     }
 
-    init(statisticsStorage: StatisticsStore = StatisticsUserDefaults(), secureVault: (any AutofillSecureVault)? = try? AutofillSecureVaultFactory.makeVault(reporter: SecureVaultReporter.shared)) {
+    init(statisticsStorage: StatisticsStore = StatisticsUserDefaults(), secureVault: (any AutofillSecureVault)? = nil) {
```

Comment:
Small optimisation here since it wasn't necessary to always initialise secureVault immediately on app launch 🙂